buildSrc/src/main/kotlin/com/getstream/sdk/chat/Dependencies.kt

```diff
@@ -30,7 +30,6 @@ private const val DRAWABLETOOLBOX_VERSION = "1.0.7"
 private const val DOKKA_VERSION = "1.4.30"
 private const val DOTENV_VERSION = "5.2.2"
 private const val ESPRESSO_VERSION = "3.3.0"
-private const val EXOPLAYER_VERSION = "2.12.1"
```

Comment:
We didn't use ExoPlayer directly anywhere, we're using it via the ExoMedia wrapper